File: AtCoder/ACP/BootCamp4b/Medium100/python/B_Choose_Integers.py
A, B, C = list(map(int, input().split()))


def gcd(a, b):
    if b == 0:
        return a
    else:
        return gcd(b, a % b)


if C % gcd(A, B) == 0:
    print('YES')
else:
    print('NO')

File: AtCoder/ACP/BootCamp4b/Medium100_2/cpp/B_Choose_Integers.cpp
#include <bits/stdc++.h>
using namespace std;

int A, B, C;

void solve() {
  int g = __gcd(A, B);
  if (C % g == 0) {
    puts("YES");
  } else {
    puts("NO");
  }
}

int main() {
  cin >> A >> B >> C;
  solve();
}